app/assets/javascripts/edit_plan.js

```diff
@@ -1,154 +1,212 @@
-$(document).on('turbolinks:load', function() {
-
-  // Max Date for Cancellation Datepicker
-  // Minimum date is current date
-  // maximum date is end of current year (I.E. 12/31/2019)
-  $("#term-date").datepicker({minDate: 0, maxDate: new Date(new Date().getFullYear(), 11, 31)});
-  $("#term-date").change(function(){
-    if( $("#agreement_action-confirm-yes").attr('checked') ){
-      $("#btn-cancel").attr("disabled", false);
-    }
+var bs4 = document.documentElement.dataset.bs4;
```

Comment:
this will work for now, but may be worth updating this to be `document.documentElement.dataset.bs4 == "true"` in case that attribute changes in the future

Reply:
https://github.com/ideacrew/enroll/pull/4327/commits/adbbf95c4afa9407a11ed2e2d1a124d8ff98d22e#diff-b41dfd5a920e57e02580f0f4255b01d17076c7434fcc521276224e1feca38555R1 thanks, done